WrightTools/data.py

```diff
@@ -2861,9 +2861,79 @@ def from_shimadzu(filepath, name=None, verbose=True):
     return data
 
 
+def from_1130(filepath, name=None, delimiter=None, verbose=True):
+    """Create a data object from a SPC-130 TCSPC an exported
+    comma-delimited (.asc) file within the SPCM 9.75 software.
```

Comment:
Docstring summary should fit on one line, additional detail may be included between the summary and Parameters section (with blank lines separating the parts)

See [Numpy](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard)

Also, [pydocstyle](http://www.pydocstyle.org) will help to ensure that the style is kept